sql/core/src/main/scala/org/apache/spark/sql/RuntimeConfig.scala

```diff
@@ -60,6 +60,16 @@ class RuntimeConfig private[sql](sqlConf: SQLConf = new SQLConf) {
     set(key, value.toString)
   }
 
+  /**
+   * Sets the given Spark runtime configuration property.
+   *
+   * @since 3.4.0
```

Comment:
non-public API doesn't need the since annotation.

Reply:
okey